redfish-core/lib/account_service.hpp

```diff
@@ -807,6 +809,85 @@ inline void
         ldapConfigInterface, "UserNameAttribute",
         std::variant<std::string>(userNameAttribute));
 }
+
+inline void acfGetMessageSetProperties(
+    const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+    const std::tuple<std::vector<uint8_t>, bool, std::string>& messageFDbus)
+{
+
+    // Get messages from call to InstallACF and add values to json
+    std::vector<uint8_t> ACFFile = std::get<0>(messageFDbus);
+    std::string decodeACFFile(ACFFile.begin(), ACFFile.end());
+    std::string encodedACFFile = crow::utility::base64encode(decodeACFFile);
+
+    bool ACFInstalled = std::get<1>(messageFDbus);
+    std::string expirationDate = std::get<2>(messageFDbus);
+
+    std::string currTime;
+    asyncResp->res.jsonValue["Oem"]["IBM"]["ACF"]["BMCCurrentDate"] = nullptr;
```

Comment:
I thought you agreed to remove this property (and require users to GET /redfish/v1/Managers/bmc property DateTime).

Reply:
Removed.